user/src/com/google/gwt/resources/rebind/context/InlineResourceContext.java

```diff
@@ -32,6 +32,7 @@ class InlineResourceContext extends StaticResourceContext {
   }
 
   @Override
+  @SuppressWarnings("checkstyle:SpaceAfterColon")
```

Comment:
For cases where we just have a single line that needs this, can we make the SuppressWarnings more specific - either just the deploy() method, or the `String encoded` local variable itself?

e.g.
```
@SuppressWarnings("checkstyle:SpaceAfterColon")
String encoded = "\"data:" + mimeType.replaceAll("\"", "\\\\\"")
    + ";base64," + base64Contents + "\"";
```

I appreciate that a lot of these cases tend to be a blanket "the whole file", but when we can make it more specific for non-test, it seems wise to do so (same as we also moved to do for errorprone suppressions).